### PR TITLE
add initial odgi tools

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
depends on: https://github.com/bioconda/bioconda-recipes/pull/21343

File: tools/odgi/build.xml [hunk 1]
```diff
@@ -1,53 +1,48 @@
...
     ]]></command>
     <inputs>
-        <param argument="--gfa" type="data" format="gf1a" label="Construct the graph from this GFA input file" />
-        <param argument="--sort" type="boolean" truevalue="--sort" falsevalue="" checked="False"
+        <param argument="--gfa" type="data" format="gfa1" label="Construct the graph from this GFA input file" />
+        <param argument="--sort" type="boolean" truevalue="--sort" falsevalue="" checked="false"
             label="Topological sort the graph"
             help="Apply generalized topological sort to the graph and set node ids to order." />
-        <param argument="--to-gfa" name="to_gfa" type="boolean" truevalue="--to-gfa" falsevalue="" checked="False"
+        <param argument="--to-gfa" type="boolean" truevalue="--to-gfa" falsevalue="" checked="false"
```
Comment (bgruening) [inline tools/odgi/build.xml:21]: hu, is the `-` really converted to `_`? I missed that.

Comment (nsoranzo) [reply tools/odgi/build.xml:21]: Yes!

Comment (nsoranzo) [reply tools/odgi/build.xml:21]: See https://docs.galaxyproject.org/en/master/dev/schema.html#id24

**Pull request merged**